### ISSUE TITLE
Secondary groups names instead of their IDs in the 'oneuser show' command output

### ISSUE THREAD

**Issue by mkutouski** [OpenNebula/one#6822]
**Description**
In would be convenient to have secondary groups names instead of their IDs in the `oneuser show` command output.
Now it looks like below:
```
oneuser show 2
USER 2 INFORMATION                                                              
ID              : 2                   
NAME            : testusr1            
GROUP           : users               
SECONDARY GROUPS: 1,100,101           
PASSWORD        : 5e884898da28047151d0e56f8dc6292773603d0d6aabbdd62a11ef721d1542d8
AUTH_DRIVER     : core                
ENABLED         : Yes             
```

but it would be more convenient to have it like that:
```
oneuser show 2
USER 2 INFORMATION                                                              
ID              : 2                   
NAME            : testusr1            
GROUP           : users               
SECONDARY GROUPS: users,group1,group2           
PASSWORD        : 5e884898da28047151d0e56f8dc6292773603d0d6aabbdd62a11ef721d1542d8
AUTH_DRIVER     : core                
ENABLED         : Yes                 
```

**Use case**
Whenever the cloud admin needs to get an information about user's secondary groups

**Interface Changes**
it does not seem like some interface changes are required. One just need to add some kind of additional command like switch either to enable secondary group names or if it will be by default then some switch might be required to have previous behavior in the sake of backward compatibility.

**Additional context**
`man oneuser` mentions `-n` switch but
1) it's not clear which command it applies for (it does not work with `list`, `show`);
2) it seems like that switch is aimed to do reverse thing: convert group/user names to their numeric format.

## Progress Status
- [ ] Code committed
- [ ] Testing - QA
- [ ] Documentation (Release notes - resolved issues, compatibility, known issues)

**Comment by mkutouski** [comment]
duplicates #1118?

**Comment by tinova** [comment]
yes, closed the older one since this description is more elaborated
